 10-define_SIZE_T_MAX.diff by Anibal Monsalve Salazar
 define SIZE_T_MAX as UINT_MAX in limits.h

--- a/libedit/readline.c	2008-04-29 16:53:01.000000000 +1000
+++ a/libedit/readline.c	2008-06-14 23:54:16.000000000 +1000
@@ -61,6 +61,10 @@ __RCSID("$NetBSD: readline.c,v 1.75 2008
 #include "readline/readline.h"
 #include "filecomplete.h"
 
+#ifndef SIZE_T_MAX
+#define SIZE_T_MAX UINT_MAX
+#endif
+
 void rl_prep_terminal(int);
 void rl_deprep_terminal(void);
 
